packages/default/scss/scrollview/_layout.scss

```diff
@@ -1,154 +1,175 @@
 @import '../common/icons';
 @import '../button/layout';
 
-$scrollview-pager-offset: 20px !default;
-$scrollview-pager-item-spacing: 20px !default;
-
 @include exports('scrollview/layout') {
-    $scrollview-page-button-size: 8px;
-    $scrollview-arrow-icon-size: 4.5em;
 
-    .k-scrollview-wrap {
+    //refactor when https://github.com/telerik/kendo-angular-scrollview/issues/130 is done
+    kendo-scrollview.k-scrollview-wrap,
+    div.k-scrollview {
         position: relative;
         overflow: hidden;
         outline: 0;
         display: block;
-    }
 
-    .k-scrollview {
-        list-style-type: none;
-        position: absolute;
-        margin: 0;
-        padding: 0;
-        width: 100%;
-        height: 100%;
-        cursor: default;
-
-        img {
-            user-select: none;
+        .k-scrollview,
+        .k-scrollview-wrap {
+            list-style-type: none;
+            position: absolute;
+            margin: 0;
+            padding: 0;
+            width: 100%;
+            height: 100%;
+            cursor: default;
+
+            img {
+                user-select: none;
+            }
+
+            & > li {
+                display: inline-block;
+                overflow: hidden;
+                position: absolute;
+                top: 0;
+                left: 0;
+            }
         }
 
-        & > li {
-            display: inline-block;
-            overflow: hidden;
+        .k-scrollview-pageable,
+        .k-scrollview-nav {
+            margin: 0 auto;
```

Comment:
Only `margin: 0;`. You already have `justify-content: center;` so no need to apply `auto`.